packages/openapi-typescript/test/transform/schema-object/string.test.ts

```diff
@@ -127,7 +127,7 @@ describe("transformSchemaObject > string", () => {
       async () => {
         const result = astToString(transformSchemaObject(given, options));
         if (want instanceof URL) {
-          expect(result).toMatchFileSnapshot(fileURLToPath(want));
+          await expect(result).toMatchFileSnapshot(fileURLToPath(want));
```

Comment:
Love all these cleanups. The test output had quite a lot of unnecessary warnings.

Does this PR clean up all of them? If so, it would be pretty great to switch these from warnings to errors in the vitest config to prevent new ones from sneaking in.

Reply:
I'm going to merge for now, but am happy to add additional rules once we find them!

Reply:
The way I typically handle this isn't in vitest, but with the [`no-floating-promises`](https://typescript-eslint.io/rules/no-floating-promises/) typescript-eslint rule.